core/win32/drwinapi/ntdll_redir.c

```diff
@@ -931,7 +938,7 @@ redirect_RtlFlsFree(IN DWORD index)
     if (!NT_SUCCESS(res))
         return res;
 
-    bitmap_mark_freed_sequence(peb->TlsBitmap->BitMapBuffer, peb->TlsBitmap->SizeOfBitMap,
+    bitmap_mark_freed_sequence(peb->FlsBitmap->BitMapBuffer, peb->FlsBitmap->SizeOfBitMap,
```

Comment:
(Hmm, so this was a bug before: I wonder if it caused any visible behavior problems.)